basket/base/tests/test_decorator.py

```diff
@@ -46,21 +47,18 @@ def test_rq_task(
         )
 
     @override_settings(MAINTENANCE_MODE=True)
-    @patch("basket.base.decorators.statsd")
-    def test_maintenance_mode_no_readonly(self, mock_statsd):
+    def test_maintenance_mode_no_readonly(self, metrics_mock):
```

Comment:
Where does the `metrics_mock` come from?

Reply:
The pytest fixture in `conftest.py`. I'm not a fan of how pytest fixtures are implicitly passed around, but once you know, you know.

Reply:
I swear I looked for a file like that before I asked 🤦‍♂️Thanks.

Reply:
It's in an odd spot, only because we have a `tests/` directory inside each Django app.